brooklyn-server/karaf/features/src/main/history/dependencies.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I have removed the file and supposedly the pom instructions that generate it. If the file keeps reappearing we can gitignore it, and ungitignore+add it later on if we need it.